FitNesseRoot/HttpTestSuite/FileServerTestSuite/CreateReadUpdateDelete/content.txt

```diff
@@ -0,0 +1,22 @@
+|script  |http browser                          |
+|set host|localhost                             |
+|set port|5000                                  |
+|get     |/new_file.txt                         |
+|ensure  |response code equals|404              |
+|set data|Some text for a new file|
+|put     |/new_file.txt                         |
+|ensure  |response code equals|201              |
+|ensure  |response header location equals|/new_file.txt|
```

Comment:
Actually, the `Location` header isn't appropriate for PUT responses because the URL is always the location. The `Location` header is only needed for `POST` requests because they create/modify a resource other than the one requested.

Reply:
:confounded:
Removed it